bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -156,6 +162,39 @@ bool ReplicaImp::validateMessage(MessageBase *msg) {
   }
 }
 
+/**
+ * validatePrePrepareMsg initiates the validation and waits for completion of validation.
+ * After validation, it routes the preprepare message as an internal message.
+ *
+ * @param msg : This is the PrePrepare message received by the replica, which is about
+ * to get handled.
+ * @return : returns nothing
+ */
+void ReplicaImp::validatePrePrepareMsg(PrePrepareMsg *&ppm) {
+  RequestThreadPool::getThreadPool().async(
+      [this](auto *mesg, auto *rpi, auto *iq) {
```

Comment:
mesg->msg
Parameter/variable names are unclear: rpi, iq, iMsgCvd

Reply:
Thanks @yuliasherman for your comment, I have incorporated the changes.